suites/pacific/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -151,20 +155,55 @@ tests:
   - test:
       name: Deploy stretch Cluster
       module: test_stretch_deployment_with_placement.py
-      polarion-id: CEPH-83573621
+      polarion-id: CEPH-83573621, CEPH-83572692
```

Comment:
@pdhiran Please include CEPH-83573625 here along with CEPH-83572692, CEPH-83573625 is marked automated in Polarion but isn't present in ceph-ci against any test case.

CEPH-83573621 can be skipped IMO as it is at least being covered somewhere else

Reply:
@pdhiran Ignore the above comment, I will take care of it in my PR